sdcm/utils/tablets/common.py

```diff
@@ -40,8 +43,13 @@ def wait_no_tablets_migration_running(node):
     time.sleep(60)  # one minute gap before checking, just to give some time to the state machine
     client = RemoteCurlClient(host="127.0.0.1:10000", endpoint="", node=node)
     LOGGER.info("Waiting for having no ongoing tablets topology operations")
-    for _ in range(3):
-        client.run_remoter_curl(method="POST", path="storage_service/quiesce_topology",
-                                params={}, timeout=3600, retry=3)
-        time.sleep(5)
-    LOGGER.info("All ongoing tablets topology operations are done")
+    try:
+        with adaptive_timeout(Operations.TABLET_MIGRATION, node, timeout=timeout):
+            for _ in range(3):
+                client.run_remoter_curl(method="POST", path="storage_service/quiesce_topology",
+                                        params={}, timeout=4*3600, retry=3)
+                time.sleep(5)
+        LOGGER.info("All ongoing tablets topology operations are done")
+    except Exception as exc:  # pylint: disable=broad-except  # noqa: BLE001
+        InfoEvent(f"Failed to wait for having no ongoing tablets topology operations. Exception: {exc.__repr__()}",
```

Comment:
isn't `adaptive_timeout` gonna raise an event if timing out or failing ?
if it doesn't raise events on failures, maybe it can extend to do so ?

Reply:
adaptive timeout raises soft timeout event, but in case of hard timeout is reached (4*3600 in this case) it is going to be raised as exception - still we want to pass this case for future node cleanup but for other cases I wouldn't change that behavior.